bindings/python/examples/03_generate_addresses.py

```diff
@@ -0,0 +1,11 @@
+import iota_client
+client = iota_client.Client()
+
+address_changed_list = client.get_addresses(
+    seed="b3d7092195c36d47133ff786d4b0a1ef2ee6a0052f6e87b6dc337935c70c531e",
```

Comment:
Although this is only an toy example, I think it might be better to let users know the correct way of using the seed. The way of using seed by directly assign the value in the codes is quite dangerous. One of the better ways is to feed the seed value from the environmental variable. The following codes provide an example of how to feed the seed from the the environmental variable called `STRONGHOLD_PASSWORD`.
```python
import os
from dotenv import load_dotenv

# Load the env variables
load_dotenv()

# Get the stronghold password
STRONGHOLD_PASSWORD = os.getenv('STRONGHOLD_PASSWORD')
```python

Reply:
Added a generic code that reads `seed` from environment variable. Also added a notice in docs